lib/netty

```diff
@@ -1 +1 @@
-Subproject commit 871ce43b1f431536e08a36d420875494ef3e9451
+Subproject commit 74627483d7bdcae3130b2ba173e08ec5f2658377
```

Comment:
Revert back to previous commit. You needed to run `git submodule update` (7462748 is older than 871ce43)